### PR TITLE
Devlogger

### PR DESCRIPTION
**Pull request by skatulich**

### CODE REVIEW

**Review by swen670group1 — DISMISSED** [review]
approve

File: js/main.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ function init(slide) {
   addResponse(slide.id, { 'test_date': testDate })
   // define user environment information: OS and browser version and add to response
   addResponse(slide.id, { 'user_environment': window.navigator['appVersion'] })
-  showControls(slide)
+  showControls(slide);
+  Logger.getInstance().addDevNote("Main init", slide, session);
```
Comment (kemaric) [inline js/main.js]: This didn't need to be done as we don't use the files in js.

File: src/app/css/reveal.css [hunk 2]
```diff
@@ -396,17 +396,17 @@ body {
...
 
-/*********************************************
- * SLIDE NUMBER
+/*********************************************
+ * SLIDE NUMBER
  *********************************************/
 .reveal .slide-number {
   position: absolute;
   display: block;
-  left: 12px;
```
Comment (kouissar) [inline src/app/css/reveal.css:407]: These changes are undoing the progress bar and page count style

File: src/app/css/reveal.css [hunk 3]
```diff
@@ -396,17 +396,17 @@ body {
...
-  bottom: 16px;
+  right: 8px;
+  bottom: 8px;
   z-index: 31;
   font-family: Helvetica, sans-serif;
-  font-size: 18px;
+  font-size: 12px;
```
Comment (kouissar) [inline src/app/css/reveal.css:411]: These changes are undoing the progress bar and page count style

**Review by kouissar — DISMISSED** [review]
some changes are undoing the progress bar and page count style. See my comments above. Also this branch has some conflicts, please reconcile with Dev. Was there any end 2 end testing done for these changes?

**Comment by kemaric** [comment]
@kouissar I resolved the conflicts and pulled in dev.

**Review by kemaric — APPROVED** [review]

**Review by kouissar — APPROVED** [review]

**Pull request merged**